rasp/base.py

```diff
@@ -16,23 +13,27 @@ def cleanup(self):
 
 
 class DefaultEngine(Engine):
-    def __init__(self, data=None, headers=None):
-        self.data = data
+    def __init__(self, headers=None):
+        self.session = self._session()
         self.headers = headers or {'User-Agent': DEFAULT_USER_AGENT}
```

Comment:
Probably should add a doc string explaining the default nature of these as well.